Hi @TomasVotruba,

I tried create custom Rector `SetList` class for laminas-servicemanager-migration, named: `Laminas\ServiceManager\Migration\Rector\Set\ValueObject\SetList`, so it need to be usable to:

```php
<?php
// rector.php
declare(strict_types=1);

use Rector\Core\Configuration\Option;
use Laminas\ServiceManager\Migration\Rector\Set\ValueObject\SetList;
use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;

return static function (ContainerConfigurator $containerConfigurator): void {
    $parameters = $containerConfigurator->parameters();
    $parameters->set(Option::PATHS, [__DIR__ . '/module']);

    $parameters->set(Option::SETS, [
        SetList::LAMINAS_SERVICEMANGER_40,
    ]);
};
```

However, when I tried, it shows that `Set` was not found:

```bash
➜  laminas-mvc-skeleton vendor/bin/rector process module/Application/src/Listener                                                                                     

                                                                                                                        
 [ERROR] Set                                                                                                            
         "/Users/samsonasik/www/laminas-mvc-skeleton/vendor/laminas/laminas-servicemanager-migration/src/Rector/Set/Valu
         eObject/../../../../config/rector/set/laminas-servicemanager-40.php" was not found                             
                                                                                                                        

 ! [NOTE] Pick one of:                                                                                                  

 * defluent
 * action-injection-to-constructor-injection
 * array-str-functions-to-static-call
 * cakephp30
 * cakephp34
 .
 .
 .
```

I re-check the path even with realpath, and the path actually correct. 

Do you have idea how to make it works? Thank you.

--- a/docs/book/v1/intro.md
+++ b/docs/book/v1/intro.md
@@ -14,12 +14,10 @@ use Laminas\ServiceManager\Migration\Rector\Set\ValueObject\SetList;
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->import(SetList::LAMINAS_SERVICEMANGER_40);
+
     $parameters = $containerConfigurator->parameters();
     $parameters->set(Option::PATHS, [__DIR__ . '/module']);
-
-    $parameters->set(Option::SETS, [
-        SetList::LAMINAS_SERVICEMANGER_40,
-    ]);
 };
 ```
 

